### ISSUE TITLE
Voiceover users cannot interact with Tree component when using VO keys

### ISSUE THREAD

**Issue by jurokapsiar** [microsoft/fluent-ui-react#1552]
### Steps
Open tree example in https://www.w3.org/TR/wai-aria-practices-1.1/examples/treeview/treeview-1/treeview-1a.html and try to navigate using VO keys

The same is reproducible in Stardust version 0.33+
### Expected Result
Navigation works

### Actual Result
Voiceover narrates Table, 0 rows, in both Chrome and Safari

We will need to find alternative structure for the tree when using tree on Mac. Also, let's file a bug on their side

### Version
0.33+ (not released yet)

**Comment by silviuaavram** [comment]
Replaced tree with list and treeitem with listitem. The navigation with VO keys works in a list-like way. You don't have the option to enter/exit the table. You just navigate list -> list items row by row -> end of list. Both list and end of list will have all the tree selected.

**Comment by jurokapsiar** [comment]
A separate set of behaviors will be created for Tree, consumer can decide, which  one to use.
